docsrc/imap/concepts/deployment/databases.rst

```diff
@@ -250,7 +250,11 @@ This is either cyrus.squat in each folder, or if you're using Xapian a single
 <userid>.xapianactive file listing active databases with tier name and number.
 
 cyrus.indexed.db is used by the Xapian search engine.  Its file type
+<<<<<<< HEAD
```

Comment:
?

Reply:
Hmm, oops - that's noise from the zeroskip rebase.  Will fix.